integr8ly/integreatly-operator
# MGDAPI-4707 - update 3scale-operator to v0.10.0-mas

# Issue link
[MGDAPI-4707](https://issues.redhat.com/browse/MGDAPI-4707)

# What
Upgrade to 3scale-operator version v0.10.0-mas

# Verification steps
- Deploy cluster
- Verify upgrading from RHOAM 1.27.0 to changes in this branch result in 3scale-operator upgrade
- Watch workload web app to determine if there is any downtime
